src/main/java/com/bookingApp/bookingApp/configuration/LoadDatabase.java

```diff
@@ -0,0 +1,97 @@
+package com.bookingApp.bookingApp.configuration;
+
+import com.bookingApp.bookingApp.model.*;
+import com.bookingApp.bookingApp.repository.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.util.ArrayList;
+import java.util.List;
+
+@Configuration
+public class LoadDatabase {
```

Comment:
to na produkcji nie powinno się tu raczej znaleźć?

Reply:
ok